docs/install/manual-install.md

```diff
@@ -128,11 +128,45 @@ As well as installing dependencies, this script compiles our CSS (using
 `bin/make_css`), installs the database schema (using `bin/update-schema`),
 and compiles any translation `.mo` files (using `commonlib/bin/gettext-makemo`).
 
+#### macOS troubleshooting
```

Comment:
We've already got a section with the heading `#### b. Mac OS X`, perhaps we should change that heading to macOS and then move this section you've added as a subsection under there. Something like `##### iii. Troubleshooting Perl module errors` or similar?

Reply:
I felt it was better to put Perl module troubleshooting in this section as it is specifically dealing with Perl modules (we assume at this point that the reader has installed the required packages in the previous section). But I should move `libpng` there, as you suggest below.